src/measure/index.js

```diff
@@ -0,0 +1,12 @@
+import { start } from '../start';
+
+/**
+ * Get the measurement diff from start
+ * @param  {String} measurement
+ * @return {Number}
+ */
+export const measure = (measurement) => {
+    const t = window.performance.timing[measurement];
+
+    return t > start() ? t - start() : 0;
```

Comment:
Can there be cases in which `t` is smaller than start?
If not, than `return t - start()` should suffice.

Reply:
`t` can be 0